tests/_data/pk.multiple.csv

```diff
@@ -3,5 +3,5 @@
 "c","b","d"
 "a","a","b"
 "a","b","c"
-"10",,"something"
-"10",,"something"
+"10","c","something"
+"10","c","something"
```

Comment:
Tady to nahrazování hodnot u současných testů nemožeme dělat, musí sa udělat vlastní test pro exasol, to je potřeba testovat, že sa to správně naloaduje a dobře proběhne dedup když je tam prázný string.

Reply:
zrusil jsem, pridal jsem kopii souboru, kde ten PK neni prazdnej